### ISSUE TITLE
[FEATURE REQUEST] Compatibility with Supplementaries

### ISSUE THREAD

**Issue by DEAFG0AT** [MrCrayfish/Backpacked#167]
Hello! Please read.

The Supplementaries mod changes the mechanics and appearance of the Wandering Trader, like so:
![2023-08-15_22 23 03_2](https://github.com/MrCrayfish/Backpacked/assets/104733108/ceb16388-9157-4d33-8cec-7df40a5feeb0)
![2023-08-15_22 23 03](https://github.com/MrCrayfish/Backpacked/assets/104733108/cc4805a0-9516-4782-99e5-bc3cd7816654)

As you can also see, these traders don't have backpacks. As such, it is impossible to unlock the Wandering Bag model.
![2023-08-15_22 24 04](https://github.com/MrCrayfish/Backpacked/assets/104733108/49b77344-de38-4aeb-a29a-7f1d92400d25)

**Comment by tad3405** [comment]
Yeah I've noticed this too and was thinking that it was just a bug on my part. Thank you for this!

**Comment by CourierNatix** [comment]
Having this issue too, various wandering traders aren't spawning with a backpack. the vanilla wandering trader seems to be a viable spawn still but also doesn't get a backpack.
